etc/docker/helm/rucio-server/README.md

```diff
@@ -0,0 +1,73 @@
+# Rucio
+
+##  Data Management for science in the Big Data era.
+
+Rucio is a project that provides services and associated libraries for allowing scientific collaborations to manage large volumes of data spread across facilities at multiple institutions and organisations. Rucio has been developed by the ATLAS. experiment. It offers advanced features, is highly scalable and modular. Rucio is a data management solution that could cover the needs of different communities in the scientific domain (e.g., HEP, astronomy, biology).
```

Comment:
There's an additional dot behind ATLAS. And I would remove the last sentence (also from the main readme.md)

Reply:
just copy/pasted the whole things from the main page. I'll change it on boths readme's